packages/contracts-bedrock/.storage-layout

```diff
@@ -89,19 +80,15 @@
 | batcherHash     | bytes32                                | 103  | 0      | 32    | src/L1/SystemConfig.sol:SystemConfig |
 | gasLimit        | uint64                                 | 104  | 0      | 8     | src/L1/SystemConfig.sol:SystemConfig |
 | _resourceConfig | struct ResourceMetering.ResourceConfig | 105  | 0      | 32    | src/L1/SystemConfig.sol:SystemConfig |
-| startBlock      | uint256                                | 106  | 0      | 32    | src/L1/SystemConfig.sol:SystemConfig |
 
 =======================
 ➡ src/L1/L1ERC721Bridge.sol:L1ERC721Bridge
 =======================
 
-| Name          | Type                                                             | Slot | Offset | Bytes | Contract                                 |
-|---------------|------------------------------------------------------------------|------|--------|-------|------------------------------------------|
-| _initialized  | uint8                                                            | 0    | 0      | 1     | src/L1/L1ERC721Bridge.sol:L1ERC721Bridge |
-| _initializing | bool                                                             | 0    | 1      | 1     | src/L1/L1ERC721Bridge.sol:L1ERC721Bridge |
-| messenger     | contract CrossDomainMessenger                                    | 0    | 2      | 20    | src/L1/L1ERC721Bridge.sol:L1ERC721Bridge |
-| __gap         | uint256[48]                                                      | 1    | 0      | 1536  | src/L1/L1ERC721Bridge.sol:L1ERC721Bridge |
-| deposits      | mapping(address => mapping(address => mapping(uint256 => bool))) | 49   | 0      | 32    | src/L1/L1ERC721Bridge.sol:L1ERC721Bridge |
+| Name     | Type                                                             | Slot | Offset | Bytes | Contract                                 |
+|----------|------------------------------------------------------------------|------|--------|-------|------------------------------------------|
+| __gap    | uint256[49]                                                      | 0    | 0      | 1568  | src/L1/L1ERC721Bridge.sol:L1ERC721Bridge |
+| deposits | mapping(address => mapping(address => mapping(uint256 => bool))) | 49   | 0      | 32    | src/L1/L1ERC721Bridge.sol:L1ERC721Bridge |
```

Comment:
Not really a diff, just a width change.

Reply:
I see that deposits is at the same slot so we are good here